Sources/OpenAPIRuntime/Base/Base64EncodedData.swift

```diff
@@ -56,7 +56,13 @@ public struct Base64EncodedData: Sendable, Hashable {
 
     /// Initializes an instance of ``Base64EncodedData`` wrapping the provided slice of bytes.
     /// - Parameter data: The underlying bytes to wrap.
+    @available(*, deprecated, renamed: "init", message: "Use the init(_ data: ArraySlice<UInt8>) instead.")
+
     public init(data: ArraySlice<UInt8>) { self.data = data }
+
+    /// Initializes an instance of ``Base64EncodedData`` wrapping the provided slice of bytes.
+    /// - Parameter data: The underlying bytes to wrap.
+    public init(_ data: ArraySlice<UInt8>) { self.data = data }
```

Comment:
```suggestion
    public init(_ data: ArraySlice<UInt8>) { self.data = data }
    
    /// Initializes an instance of ``Base64EncodedData`` wrapping the provided sequence of bytes.
    /// - Parameter data: The underlying bytes to wrap.
    public init(_ data: some Sequence<UInt8>) { self.init(ArraySlice(data)) }
```

With this, adopters can easily also pass `[UInt8]`, `ByteBuffer.readableBytesView`, and `Data` itself – without an explicit wrapping in `ArraySlice`.